### ISSUE TITLE
Why don't use the poperty 'Name' of HttpMethodAttribute (HttpGet/HttpPost...) to generate swagger operationId ?

### ISSUE THREAD

**Issue by nikobiker12** [RSuter/NSwag#1152]
Today, swagger operationId is generated 
- either from using convention [_ControllerName_]_[_OperationName_]
- or using NSwagOperationAttribute from NSwag.Annotations

Why do not consider using the property 'Name' of HttpMethodAttribute (derived class are well knwown  HttpGet/HttpPost...) already used on  to specify HTTP Method for operations ?

Regards

**Comment by MelGrubb** [comment]
I agree with this suggestion. The more information can come from standard attributes the better.

**Comment by Rickybarb5** [comment]
Any news on this enhancement?

**Comment by RSuter** [comment]
For what is this Name property used for originally?

My biggest concern with that is that this might be a breaking change for some users and it might be used for something else and you will be forced to used.. or put this behind a setting/flag?
